src/lib/Service/ExportNotificationService.php

```diff
@@ -42,7 +46,7 @@ public function createNotification(array $options): Notification
         $notification->events = $options['events'];
         $notification->licenseKey = $options['licenseKey'];
         $notification->customerId = (int)$options['customerId'];
-        $notification->transaction = (new \DateTime())->format('YmdHis') . rand(111, 999);
+        $notification->transaction = (new \DateTime())->format('YmdHis') . random_int(111, 999);
```

Comment:
```suggestion
        $notification->transaction = date('YmdHis') . random_int(111, 999);
```

Instantiating a `DateTime` object is unnecessary. Calling `date()` is equivalent.